### PR TITLE
[PROTOCOL-419] [L08] Naming hinders code understanding and readability

### PR DESCRIPTION
**Pull request by sypeer**
It is:
- Renamed names of the functions and contracts for better readability

### CODE REVIEW

**Review by passabilities — CHANGES_REQUESTED** [review]
* Keep the naming of `supplyToDebt` and inverse the ratio for the calculation.
* Rename all interfaces to be prefixed with **I** (ex: `IBaseProxy.sol`) instead of the postfix **Interface** (ex: `BaseProxyInterface.sol`)

**Comment by sypeer** [comment]
> * Keep the naming of `supplyToDebt` and inverse the ratio for the calculation.
> * Rename all interfaces to be prefixed with **I** (ex: `IBaseProxy.sol`) instead of the postfix **Interface** (ex: `BaseProxyInterface.sol`)

Done dawg

**Review by salazarguille — CHANGES_REQUESTED** [review]
Great Job Syed, I know it was a heavy change.

Please, verify the CI, and check if the integration tests are  passing.

File: test/base/MarketsStateGetDebtRatioForTest.js [hunk 1]
```diff
@@ -2,17 +2,17 @@
...
 
 // Mock contracts
 const Mock = artifacts.require("./mock/util/Mock.sol");
 
 // Smart contracts
 const MarketsState = artifacts.require("./base/MarketsState.sol");
 
-contract('MarketsStateGetSupplyToDebtForTest', function (accounts) {
-    const settingsInterfaceEncoder = new SettingsInterfaceEncoder(web3);
+contract('MarketsStateGetDebtToSupplyForTest', function (accounts) {
```
Comment (hensha256) [inline test/base/MarketsStateGetDebtRatioForTest.js:14]: Most of this PR has the new wording has 'Debt Ratio' not 'DebtToSupply'?

File: test/base/MarketsStateGetSupplyToDebtTest.js [hunk 2]
```diff
@@ -12,8 +12,8 @@ const ERC20Mock = artifacts.require("./mock/token/ERC20Mock.sol");
 // Smart contracts
 const MarketsState = artifacts.require("./base/MarketsState.sol");
 
-contract('MarketsStateGetSupplyToDebtTest', function (accounts) {
-    const settingsInterfaceEncoder = new SettingsInterfaceEncoder(web3);
+contract('MarketsStateGetDebtToSupplyTest', function (accounts) {
```
Comment (hensha256) [inline test/base/MarketsStateGetSupplyToDebtTest.js:15]: Same here

**Review by hensha256 — COMMENTED** [review]
Well done @sypeer

**Pull request closed without merging**